src/main/webapp/app/shared/model/firebase/firebase.model.ts

```diff
@@ -80,7 +80,7 @@ export const TI_NAME: { [key in TI_TYPE]: string } = {
 };
 
 // We should convert boolean strings to actual booleans
-export type BoolString = 'false' | 'true';
+export type BoolString = 'false' | 'true' | boolean;
```

Comment:
The type is for a string boolean though.

Reply:
Will change `name_comments`' `resolved` field so that it is of type `BoolString | boolean` instead